smallrye-reactive-messaging-aws-sns/pom.xml

```diff
@@ -0,0 +1,70 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.smallrye.reactive</groupId>
+    <artifactId>smallrye-reactive-messaging</artifactId>
+    <version>1.0.4-SNAPSHOT</version>
+  </parent>
+  <properties>
+  	<vertx.version>3.8.0</vertx.version>
+  </properties>
+  <artifactId>smallrye-reactive-messaging-aws-sns</artifactId>
+  <name>Smallrye Reactive Messaging for AWS SNS</name>
+  <description>A module for Smallrye reactive messaging integration with AWS Simple Notification Service (SNS).</description>
+    <dependencyManagement>
+    <dependencies>
+      <dependency>
+	      <groupId>software.amazon.awssdk</groupId>
+	      <artifactId>bom</artifactId>
+	      <version>2.7.13</version>
+	      <type>pom</type>
+	      <scope>import</scope>
+	    </dependency>
+    </dependencies>
+  </dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>smallrye-reactive-messaging-provider</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>testcontainers</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.smallrye</groupId>
+      <artifactId>smallrye-config-1.3</artifactId>
+      <version>1.0.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>software.amazon.awssdk</groupId>
+      <artifactId>sns</artifactId>
+	</dependency>
+	<dependency>
+      <groupId>com.amazonaws</groupId>
+      <artifactId>aws-java-sdk-sns</artifactId>
+      <version>1.11.600</version>
```

Comment:
why do we need the version here? isn't it in the BOM?

Reply:
BOM is for AWS SDK Java 2. not all features of AWS SDK JAVA 1 has been shifted to 2. that's why I am using both SDK (and it is safe to have them both). once they shift all features to SDK v2 I will remove SDK v1.